quipucords/scanner/network/processing/karaf.py

```diff
@@ -124,15 +125,18 @@ def process(output, dependencies=None):
         for item in output['results']:
             result = {}
             item_name = item['item']
-            if item.get('rc', True):
-                pass
-            else:
-                if item['stdout'] != '':
-                    value = str(item['stdout']).splitlines()
-                    result['install_home'] = item_name
-                    result['version'] = list(set(value))
-            if result:
-                results.append(result)
+            if item_name:
```

Comment:
Should we change line 128 to a item.get('item')? I realized you didn't add that code but I think we should move away for dict['key'].